### DISCUSSION TITLE
Actions

### DISCUSSION THREAD

**Discussion by CloudSVC** [community/community#53531] [Mobile] [answered]
why not Actions at mobile apps

**Comment by stevepopovich** [comment, marked as answer]
Hi @CloudSVC we actually do have support for Actions on mobile. Find more info here: https://github.blog/2023-01-20-bringing-github-actions-to-github-mobile/

Are there things specifically that you think we are missing? Thanks for writing in!

**Reply by CloudSVC** [reply]
ok, thank you.

**Comment by kakar930** [comment]
53531

**Comment by kakar930** [comment]
Hi @CloudSVC we actually do have support for Actions on mobile. Find more info here: https://github.blog/2023-01-20-bringing-github-actions-to-github-mobile/

Are there things specifically that you think we are missing? Thanks for writing in!

**Comment by bgu6667** [comment]
> Hi @CloudSVC we actually do have support for Actions on mobile. Find more info here: https://github.blog/2023-01-20-bringing-github-actions-to-github-mobile/
> 
> 
> 
> Are there things specifically that you think we are missing? Thanks for writing in!
